### DISCUSSION TITLE
Custom sort

### DISCUSSION THREAD

**Discussion by Lukeomichael** [Abrynos/ShoppingList#124] [Ideas]
I love the simplicity of this app. What I suggest is to be able to have a drag function so you can custom sort the shipping list. That's because I know what is in the Isles and I like ordering the shopping list so it aligns with the Isles

**Comment by Abrynos** [comment]
> I love the simplicity of this app

First and foremost: Thank you very much.

> drag function so you can custom sort the shipping list

This ties in with #117.

> custom sort

This is a bit more complicated since it would require modification of the database scheme and a way to make it work with button-navigation (as mentioned in #117, whether buttons or gestures are used should be determined by the system settings).
